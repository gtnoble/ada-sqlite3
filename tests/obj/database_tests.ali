V "GNAT Lib v14"
A -O0
A -gnatA
A -gnatwa
A -gnatVa
A -gnatQ
A -gnat12
A -gnato
A -gnata
A -g
A -mtune=generic
A -march=x86-64
P SS ZX

RN
RV NO_ACCESS_SUBPROGRAMS
RV NO_ALLOCATORS
RV NO_DISPATCH
RV NO_DISPATCHING_CALLS
RV NO_EXCEPTION_HANDLERS
RV NO_EXCEPTIONS
RV NO_IO
RV NO_LOCAL_ALLOCATORS
RV NO_NESTED_FINALIZATION
RV NO_SECONDARY_STACK
RV NO_STANDARD_STORAGE_POOLS
RV NO_DEFAULT_INITIALIZATION
RV NO_DYNAMIC_SIZED_OBJECTS
RV NO_IMPLEMENTATION_PRAGMAS
RV NO_ELABORATION_CODE

U database_tests%b	database_tests.adb	8f43bc01 OO PF PK
W ada%s			ada.ads			ada.ali
W ada.directories%s	a-direct.adb		a-direct.ali
W ada.exceptions%s	a-except.adb		a-except.ali
Z ada.strings.text_buffers%s  a-sttebu.adb	a-sttebu.ali
Z ada.tags%s		a-tags.adb		a-tags.ali
Z ada.unchecked_conversion%s
W ada_sqlite3%s		ada_sqlite3.adb		ada_sqlite3.ali
W aunit%s		aunit.adb		aunit.ali
W aunit.assertions%s	aunit-assertions.adb	aunit-assertions.ali
W aunit.memory%s	aunit-memory.adb	aunit-memory.ali
Z aunit.memory.utils%s	aunit-memory-utils.adb	aunit-memory-utils.ali  ED
Z aunit.simple_test_cases%s  aunit-simple_test_cases.adb  aunit-simple_test_cases.ali
W aunit.test_caller%s	aunit-test_caller.adb	aunit-test_caller.ali  ED
Z aunit.test_fixtures%s	aunit-test_fixtures.adb	aunit-test_fixtures.ali
Z system%s		system.ads		system.ali
Z system.finalization_masters%s  s-finmas.adb	s-finmas.ali
Z system.pool_global%s	s-pooglo.adb		s-pooglo.ali
Z system.put_images%s	s-putima.adb		s-putima.ali
Z system.secondary_stack%s  s-secsta.adb	s-secsta.ali
Z system.soft_links%s	s-soflin.adb		s-soflin.ali
Z system.standard_library%s  s-stalib.adb	s-stalib.ali
Z system.storage_elements%s  s-stoele.ads	s-stoele.ali
Z system.storage_pools%s  s-stopoo.adb		s-stopoo.ali
Z system.storage_pools.subpools%s  s-stposu.adb	s-stposu.ali

U database_tests%s	database_tests.ads	2f904e8a EE OO PF PK
Z ada.strings.text_buffers%s  a-sttebu.adb	a-sttebu.ali
Z ada.tags%s		a-tags.adb		a-tags.ali
W aunit%s		aunit.adb		aunit.ali
W aunit.test_fixtures%s	aunit-test_fixtures.adb	aunit-test_fixtures.ali
W aunit.test_suites%s	aunit-test_suites.adb	aunit-test_suites.ali
Z system%s		system.ads		system.ali
Z system.put_images%s	s-putima.adb		s-putima.ali
Z system.soft_links%s	s-soflin.adb		s-soflin.ali

D ada.ads		20240801081713 76789da1 ada%s
D a-calend.ads		20240801081713 158a5463 ada.calendar%s
D a-charac.ads		20240801081713 2d3ec45b ada.characters%s
D a-chlat1.ads		20240801081713 e0d72e76 ada.characters.latin_1%s
D a-direct.ads		20240801081713 467e87a1 ada.directories%s
D a-except.ads		20240801081713 e7970cd9 ada.exceptions%s
D a-finali.ads		20240801081713 bf4f806b ada.finalization%s
D a-inteio.ads		20240801081713 f64b89a4 ada.integer_text_io%s
D a-ioexce.ads		20240801081713 40018c65 ada.io_exceptions%s
D a-stream.ads		20240801081713 119b8fb3 ada.streams%s
D a-string.ads		20240801081713 90ac6797 ada.strings%s
D a-strmap.ads		20240801081713 1f91ebb6 ada.strings.maps%s
D a-sttebu.ads		20240801081713 f1ad67a2 ada.strings.text_buffers%s
D a-strunb.ads		20240801081713 98c60381 ada.strings.unbounded%s
D a-stuten.ads		20240801081713 c6ced0ae ada.strings.utf_encoding%s
D a-tags.ads		20240801081713 9eaa38c6 ada.tags%s
D a-textio.ads		20240801081713 34ef47de ada.text_io%s
D a-tiinio.ads		20240801081713 76699c8f ada.text_io.integer_io%s
D a-unccon.ads		20240801081713 0e9b276f ada.unchecked_conversion%s
D ada_containers.ads	20240312145141 11329e00 ada_containers%s
D ada_containers-aunit_lists.ads  20240312145141 d9ebc89a ada_containers.aunit_lists%s
D ada_sqlite3.ads	20250302212249 64558577 ada_sqlite3%s
D aunit.ads		20240312145141 76cdf7c6 aunit%s
D aunit-assertions.ads	20240312145141 b7f9ecd6 aunit.assertions%s
D aunit-io.ads		20240312145141 fb5ed18b aunit.io%s
D aunit-memory.ads	20240312145141 a0315146 aunit.memory%s
D aunit-memory-utils.ads  20240312145141 5b1e3d11 aunit.memory.utils%s
D aunit-memory-utils.adb  20240312145141 c00242e3 aunit.memory.utils%b
D aunit-options.ads	20240312145141 6e5a02a6 aunit.options%s
D aunit-simple_test_cases.ads  20240312145141 4111c32f aunit.simple_test_cases%s
D aunit-test_caller.ads	20240312145141 956cfce4 aunit.test_caller%s
D aunit-test_caller.adb	20240312145141 0f783ccd aunit.test_caller%b
D aunit-test_filters.ads  20240312145141 f10b2127 aunit.test_filters%s
D aunit-test_fixtures.ads  20240312145141 faadeab5 aunit.test_fixtures%s
D aunit-test_results.ads  20240312145141 b44de7dc aunit.test_results%s
D aunit-test_suites.ads	20240312145141 6821dc24 aunit.test_suites%s
D aunit-tests.ads	20240312145141 1da11d49 aunit.tests%s
D aunit-time_measure.ads  20240312145141 d9c247b0 aunit.time_measure%s
D database_tests.ads	20250303005458 cbd18fdd database_tests%s
D database_tests.adb	20250303013409 a3bb2de1 database_tests%b
D gnat.ads		20240801081713 b5988c27 gnat%s
D g-souinf.ads		20240801081713 470146ee gnat.source_info%s
D interfac.ads		20240801081713 9111f9c1 interfaces%s
D i-cstrea.ads		20240801081713 ffd01b9d interfaces.c_streams%s
D system.ads		20240801081714 14286b0f system%s
D s-atocou.ads		20240801081713 dc18efc3 system.atomic_counters%s
D s-crtl.ads		20240801081713 beb39b9e system.crtl%s
D s-exctab.ads		20240801081713 91bef6ef system.exception_table%s
D s-ficobl.ads		20240801081713 dc5161d4 system.file_control_block%s
D s-finmas.ads		20240801081713 28987ac5 system.finalization_masters%s
D s-finroo.ads		20240801081713 4ff27390 system.finalization_root%s
D s-memory.ads		20240801081713 18949c1a system.memory%s
D s-parame.ads		20240801081713 3597fc11 system.parameters%s
D s-pooglo.ads		20240801081713 91708d21 system.pool_global%s
D s-putima.ads		20240801081713 17291fe4 system.put_images%s
D s-retsta.ads		20240801081713 0f6b06cb system.return_stack%s
D s-secsta.ads		20240801081713 578279f5 system.secondary_stack%s
D s-soflin.ads		20240801081713 df77073e system.soft_links%s
D s-stache.ads		20240801081713 0b81c1fe system.stack_checking%s
D s-stalib.ads		20240801081713 1c9580f6 system.standard_library%s
D s-stalib.adb		20240801081713 eb464a47 system.standard_library%b
D s-stoele.ads		20240801081713 ccded4e8 system.storage_elements%s
D s-stopoo.ads		20240801081713 e9fa2dd8 system.storage_pools%s
D s-stposu.ads		20240801081713 e251976f system.storage_pools.subpools%s
D s-stratt.ads		20240801081713 a1861007 system.stream_attributes%s
D s-traent.ads		20240801081713 c81cbf8c system.traceback_entries%s
D s-unstyp.ads		20240801081713 fa2a7f59 system.unsigned_types%s
D s-wchcon.ads		20240801081714 d9032363 system.wch_con%s
G a e
G c b b b [b database_tests 15 1 none]
G c Z s b [test_open_close database_tests 19 14 none]
G c Z s b [test_in_memory_database database_tests 20 14 none]
G c Z s b [test_execute database_tests 21 14 none]
G c Z s b [test_last_insert_row_id database_tests 22 14 none]
G c Z s b [test_changes database_tests 23 14 none]
G c Z s b [test_version database_tests 24 14 none]
G c Z s b [test_invalid_database database_tests 25 14 none]
G c Z s b [test_invalid_sql database_tests 26 14 none]
G c Z s b [suite database_tests 29 13 none]
G c Z s s [testDF database_tests 16 9 none]
G c Z s s [testIP database_tests 16 9 none]
G c Z b b [remove_test_db database_tests 24 14 none]
G c Z b b [create database_tests__caller 85 13 219_4]
G c Z b b [create database_tests__caller 91 14 219_4]
G c Z b b [name database_tests__caller 98 13 219_4]
G c Z b b [run_test database_tests__caller 101 14 219_4]
G c Z b b [set_up database_tests__caller 104 14 219_4]
G c Z b b [tear_down database_tests__caller 107 14 219_4]
G c Z b b [test_caseDF database_tests__caller 115 9 219_4]
G c Z b b [test_caseIP database_tests__caller 115 9 219_4]
G c Z b b [new_fixtureR database_tests__caller__new_fixtureGP2498 38 13 219_4]
G c Z b b [new_fixture database_tests__caller 38 13 219_4]
G r i none [b database_tests 15 1 none] [gen_alloc aunit__memory__utils 40 13 none]
G r i none [b database_tests 15 1 none] [test_caller aunit 78 15 none]
G r s test_fixture [b database_tests 15 1 none] [test_fixtureIP aunit__test_fixtures 88 9 none]
G r s database [test_open_close database_tests 19 14 none] [databaseIP ada_sqlite3 211 9 none]
G r c none [test_open_close database_tests 19 14 none] [open ada_sqlite3 84 14 none]
G r c none [test_open_close database_tests 19 14 none] [is_open ada_sqlite3 93 13 none]
G r c none [test_open_close database_tests 19 14 none] [assert aunit__assertions 48 14 none]
G r c none [test_open_close database_tests 19 14 none] [close ada_sqlite3 90 14 none]
G r s database [test_in_memory_database database_tests 20 14 none] [databaseIP ada_sqlite3 211 9 none]
G r c none [test_in_memory_database database_tests 20 14 none] [open ada_sqlite3 84 14 none]
G r c none [test_in_memory_database database_tests 20 14 none] [is_open ada_sqlite3 93 13 none]
G r c none [test_in_memory_database database_tests 20 14 none] [assert aunit__assertions 48 14 none]
G r c none [test_in_memory_database database_tests 20 14 none] [execute ada_sqlite3 96 14 none]
G r s statement [test_in_memory_database database_tests 20 14 none] [statementIP ada_sqlite3 218 9 none]
G r c none [test_in_memory_database database_tests 20 14 none] [prepare ada_sqlite3 113 14 none]
G r c none [test_in_memory_database database_tests 20 14 none] [step ada_sqlite3 128 13 none]
G r c none [test_in_memory_database database_tests 20 14 none] [column_int ada_sqlite3 182 13 none]
G r c none [test_in_memory_database database_tests 20 14 none] [finalize_statement ada_sqlite3 119 14 none]
G r c none [test_in_memory_database database_tests 20 14 none] [close ada_sqlite3 90 14 none]
G r s database [test_execute database_tests 21 14 none] [databaseIP ada_sqlite3 211 9 none]
G r c none [test_execute database_tests 21 14 none] [open ada_sqlite3 84 14 none]
G r c none [test_execute database_tests 21 14 none] [execute ada_sqlite3 96 14 none]
G r s statement [test_execute database_tests 21 14 none] [statementIP ada_sqlite3 218 9 none]
G r c none [test_execute database_tests 21 14 none] [prepare ada_sqlite3 113 14 none]
G r c none [test_execute database_tests 21 14 none] [step ada_sqlite3 128 13 none]
G r c none [test_execute database_tests 21 14 none] [assert aunit__assertions 48 14 none]
G r c none [test_execute database_tests 21 14 none] [finalize_statement ada_sqlite3 119 14 none]
G r c none [test_execute database_tests 21 14 none] [close ada_sqlite3 90 14 none]
G r s database [test_last_insert_row_id database_tests 22 14 none] [databaseIP ada_sqlite3 211 9 none]
G r c none [test_last_insert_row_id database_tests 22 14 none] [open ada_sqlite3 84 14 none]
G r c none [test_last_insert_row_id database_tests 22 14 none] [execute ada_sqlite3 96 14 none]
G r c none [test_last_insert_row_id database_tests 22 14 none] [last_insert_row_id ada_sqlite3 101 13 none]
G r c none [test_last_insert_row_id database_tests 22 14 none] [assert aunit__assertions 48 14 none]
G r c none [test_last_insert_row_id database_tests 22 14 none] [close ada_sqlite3 90 14 none]
G r s database [test_changes database_tests 23 14 none] [databaseIP ada_sqlite3 211 9 none]
G r c none [test_changes database_tests 23 14 none] [open ada_sqlite3 84 14 none]
G r c none [test_changes database_tests 23 14 none] [execute ada_sqlite3 96 14 none]
G r c none [test_changes database_tests 23 14 none] [changes ada_sqlite3 104 13 none]
G r c none [test_changes database_tests 23 14 none] [assert aunit__assertions 48 14 none]
G r c none [test_changes database_tests 23 14 none] [close ada_sqlite3 90 14 none]
G r c none [test_version database_tests 24 14 none] [version ada_sqlite3 107 13 none]
G r c none [test_version database_tests 24 14 none] [assert aunit__assertions 48 14 none]
G r s database [test_invalid_database database_tests 25 14 none] [databaseIP ada_sqlite3 211 9 none]
G r c none [test_invalid_database database_tests 25 14 none] [open ada_sqlite3 84 14 none]
G r c none [test_invalid_database database_tests 25 14 none] [assert aunit__assertions 48 14 none]
G r s database [test_invalid_sql database_tests 26 14 none] [databaseIP ada_sqlite3 211 9 none]
G r c none [test_invalid_sql database_tests 26 14 none] [open ada_sqlite3 84 14 none]
G r c none [test_invalid_sql database_tests 26 14 none] [execute ada_sqlite3 96 14 none]
G r c none [test_invalid_sql database_tests 26 14 none] [assert aunit__assertions 48 14 none]
G r c none [test_invalid_sql database_tests 26 14 none] [close ada_sqlite3 90 14 none]
G r s test_suite [suite database_tests 29 13 none] [test_suiteIP aunit__test_suites 82 9 none]
G r i none [suite database_tests 29 13 none] [gen_alloc aunit__memory__utils 40 13 none]
G r s test_case [suite database_tests 29 13 none] [test_caseIP aunit__simple_test_cases 75 9 none]
G r c none [suite database_tests 29 13 none] [format aunit 44 13 none]
G r c none [suite database_tests 29 13 none] [add_test aunit__test_suites 48 14 none]
G r s test_fixture [testIP database_tests 16 9 none] [test_fixtureIP aunit__test_fixtures 88 9 none]
G r i none [create database_tests__caller 85 13 219_4] [gen_alloc aunit__memory__utils 40 13 none]
G r s test_case [create database_tests__caller 85 13 219_4] [test_caseIP aunit__simple_test_cases 75 9 none]
G r c none [create database_tests__caller 85 13 219_4] [format aunit 44 13 none]
G r c none [create database_tests__caller 91 14 219_4] [format aunit 44 13 none]
G r c none [run_test database_tests__caller 101 14 219_4] [copy_id aunit__assertions 125 14 none]
G r c none [set_up database_tests__caller 104 14 219_4] [set_up aunit__test_fixtures 80 14 none]
G r c none [tear_down database_tests__caller 107 14 219_4] [tear_down aunit__test_fixtures 83 14 none]
G r s test_case [test_caseIP database_tests__caller 115 9 219_4] [test_caseIP aunit__simple_test_cases 75 9 none]
G r s test_fixture [new_fixtureR database_tests__caller__new_fixtureGP2498 38 13 219_4] [test_fixtureIP aunit__test_fixtures 88 9 none]
X 1 ada.ads
16K9*Ada 20e8 40|9r6 10r6 26r10 27r10
X 5 a-direct.ads
76K13*Directories 524e20 40|9w10 26r14 27r14
147U14*Delete_File 40|27s26
310V13*Exists{boolean} 40|26s26
X 6 a-except.ads
44K13*Exceptions 309e19 40|10w10
X 21 ada_containers-aunit_lists.ads
53R9*Cursor
X 22 ada_sqlite3.ads
12K9*Ada_Sqlite3 225e16 40|13w6 18r8
15I9*Result_Code<integer> 40|67r19 96r19
47i4*ROW{15I9} 40|71r27 105r25
48i4*DONE{15I9} 40|103r32
51X4*SQLite_Error 40|186r15 207r15
68M9*Open_Flag
70m4*OPEN_READWRITE{68M9} 40|40r31 57r29 86r29 124r29 149r29 184r52 201r29
71m4*OPEN_CREATE{68M9} 40|40r49 57r47 86r47 124r47 149r47 201r47
81R9*Database<7|50R9> 214e14 40|34r12 54r12 83r12 121r12 146r12 179r12 197r12
84U14*Open 40|40s7 57s7 86s7 124s7 149s7 184s10 201s7
90U14*Close 40|44s7 77s7 115s7 140s7 164s7 215s7
93V13*Is_Open{boolean} 40|41s15 45s19 58s15
96U14*Execute 40|61s7 62s7 89s7 90s7 91s7 127s7 128s7 134s7 152s7 153s7 154s7
. 155s7 158s7 205s10
101V13*Last_Insert_Row_ID{long_integer} 40|131s15 137s15
104V13*Changes{integer} 40|161s15
107V13*Version{string} 40|170s32
110R9*Statement<7|50R9> 221e14 40|66r17 95r17
113U14*Prepare 40|69s10 99s10
119U14*Finalize_Statement 40|73s10 111s10
128V13*Step{15I9} 40|70s20 102s23
182V13*Column_Int{integer} 40|72s18
X 23 aunit.ads
34K9*AUnit 48e10 39|9w6 10r6 11r6 16r21 29r26 40|11r6 12r6 17r8 219r26 221r26
. 222r25 223r14
X 24 aunit-assertions.ads
44K15*Assertions 155e21 40|11w12 17r14
48U14*Assert 40|41s7 45s7 58s7 71s10 72s10 110s10 131s7 137s7 161s7 173s7
. 191s7 212s7
101U14*Init_Test
104U14*Clear_Failures
107V13*Has_Failures{boolean}
110R9*Failure_Iter<21|53R9[145]>
113V13*First_Failure{110R9}
X 31 aunit-test_caller.ads
78k15*Test_Caller 121e22 40|12w12 219r32
80R9*Test_Case<30|44R9>
81P9*Test_Case_Access(80R9[40|219])
85V13*Create{81P9[40|219]} 40|225s31[219] 226s31[219] 227s31[219] 228s31[219]
. 229s31[219] 230s31[219] 231s31[219]
112P9 Fixture_Access(39|16R9)
X 32 aunit-test_caller.adb
38V13 New_Fixture[27|40]{31|112P9[40|219]} 28|38b13[40|219]
66P12 Access_Type(31|80R9[40|219])
67V16 Alloc[27|40]{66P12[40|219]} 28|38b13[40|219]
X 34 aunit-test_fixtures.ads
76K15*Test_Fixtures 90e24 39|10w12 16r27
78R9*Test_Fixture<24|96R9> 88e67 39|16r41
80U14*Set_Up
83U14*Tear_Down
X 36 aunit-test_suites.ads
41K15*Test_Suites 86e22 39|11w12 29r32 40|221r32 222r31 223r20
43R9*Test_Suite<37|38R9> 84e14 40|223r32
44P9*Access_Test_Suite(43R9) 39|29r44 40|221r44 222r43
48U14*Add_Test 40|225s14 226s14 227s14 228s14 229s14 230s14 231s14
X 39 database_tests.ads
13K9*Database_Tests 31l5 31e19 40|15b14 238l5 238t19
16R9*Test<34|78R9> 24|101p14 104p14 107p13 113p13 34|80p14 83p14 39|16e70
. 19p14 19r42 20p14 20r50 21p14 21r39 22p14 22r50 23p14 23r39 24p14 24r39
. 25p14 25r48 26p14 26r43 40|32r42 52r50 81r39 119r50 144r39 168r39 177r48
. 195r43 219r45
19U14*Test_Open_Close 19=31 40|32b14 49l8 49t23 225r58
19r31 T{16R9} 40|32b31 33r28
20U14*Test_In_Memory_Database 20=39 40|52b14 78l8 78t31 226r66
20r39 T{16R9} 40|52b39 53r28
21U14*Test_Execute 21=28 40|81b14 116l8 116t20 227r55
21r28 T{16R9} 40|81b28 82r28
22U14*Test_Last_Insert_Row_ID 22=39 40|119b14 141l8 141t31 228r66
22r39 T{16R9} 40|119b39 120r28
23U14*Test_Changes 23=28 40|144b14 165l8 165t20 229r55
23r28 T{16R9} 40|144b28 145r28
24U14*Test_Version 24=28 40|168b14 174l8 174t20 230r55
24r28 T{16R9} 40|168b28 169r28
25U14*Test_Invalid_Database 25=37 40|177b14 192l8 192t29 231r64
25r37 T{16R9} 40|177b37 178r28
26U14*Test_Invalid_SQL 26=32 40|195b14 216l8 216t24
26r32 T{16R9} 40|195b32 196r28
29V13*Suite{36|44P9} 40|221b13 236l8 236t13
X 40 database_tests.adb
21a4 Test_DB_File{string} 26r34 27r39 40r17
24U14 Remove_Test_DB 24b14 29l8 29t22 37s7 48s7
34r7 DB{22|81R9} 40m13 40r13 41r24 44m14 44r14 45r28
54r7 DB{22|81R9} 57m13 57r13 58r24 61m16 61r16 62m16 62r16 69m25 69r25 77m14
. 77r14
66r10 Stmt{22|110R9} 69m19 69r19 70m26 70r26 72r30 73m30 73r30
67i10 Result{22|15I9} 70m10 71r18
83r7 DB{22|81R9} 86m13 86r13 89m16 89r16 90m16 90r16 91m16 91r16 99m25 99r25
. 115m14 115r14
95r10 Stmt{22|110R9} 99m19 99r19 102m29 102r29 111m30 111r30
96i10 Result{22|15I9} 102m13 103r23 105r16
97i10 Count{integer} 106m16 106r25 110r18
121r7 DB{22|81R9} 124m13 124r13 127m16 127r16 128m16 128r16 131r35 134m16
. 134r16 137r35 140m14 140r14
146r7 DB{22|81R9} 149m13 149r13 152m16 152r16 153m16 153r16 154m16 154r16
. 155m16 155r16 158m16 158r16 161r24 164m14 164r14
170a7 Ver{string} 173r15
179r7 DB{22|81R9} 184m16 184r16
180b7 Exception_Raised{boolean} 187m13 191r15
197r7 DB{22|81R9} 201m13 201r13 205m19 205r19 215m14 215r14
198b7 Exception_Raised{boolean} 208m13 212r15
219K12 Caller[31|78] 225r24 226r24 227r24 228r24 229r24 230r24 231r24
222p7 Result{36|44P9} 225r7 226r7 227r7 228r7 229r7 230r7 231r7 235r14
X 45 system.ads
67M9*Address
X 52 s-memory.ads
51V13*Alloc{45|67M9} 101i<c,__gnat_malloc>22
66U14*Free 102i<c,__gnat_free>22
74V13*Realloc{45|67M9} 103i<c,__gnat_realloc>22
X 61 s-stalib.adb
87U14 Break_Start 88i<c,__gnat_break_start>22

