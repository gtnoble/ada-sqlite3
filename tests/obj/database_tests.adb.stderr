database_tests.adb:10:09: warning: unit "Ada.Exceptions" is not referenced [-gnatwu]
